code/modules/mob/living/basic/vermin/mouse.dm

```diff
@@ -296,17 +296,38 @@
 	grind_results = list(/datum/reagent/blood = 20, /datum/reagent/consumable/liquidgibs = 5)
 	decomp_req_handle = TRUE
 	ant_attracting = FALSE
+	var/body_color = "gray"
+	var/critter_type = /mob/living/basic/mouse
 	decomp_type = /obj/item/food/deadmouse/moldy
```

Comment:
new bars should go after vars declared on the parent, so both of the new ones you add should be after `decomp_type`. also would be nice if you could add a DMDoc for statistical purposes